certified-connectors/Cognite Data Fusion/apiDefinition.swagger.json

```diff
@@ -0,0 +1,15694 @@
+            "in": "body",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "items": {
+                  "type": "array",
+                  "minItems": 1,
+                  "maxItems": 100,
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "space": {
+                        "type": "string",
+                        "minLength": 1,
+                        "maxLength": 43,
+                        "pattern": "^[a-zA-Z][a-zA-Z0-9_-]{0,41}[a-zA-Z0-9]?$",
+                        "description": "space"
+                      },
+                      "externalId": {
+                        "type": "string",
+                        "minLength": 1,
+                        "maxLength": 255,
+                        "pattern": "^[a-zA-Z]([a-zA-Z0-9_]{0,253}[a-zA-Z0-9])?$",
+                        "description": "externalId"
+                      },
+                      "version": {
+                        "type": "string",
+                        "minLength": 1,
+                        "maxLength": 43,
+                        "pattern": "^[a-zA-Z0-9]([.a-zA-Z0-9_-]{0,41}[a-zA-Z0-9])?$",
+                        "description": "version"
+                      }
+                    }
+                  },
+                  "description": "items"
+                }
+              },
+              "default": {
+                "items": [
+                  {
+                    "space": "qt45rYd8TMaMn",
+                    "externalId": "m",
+                    "version": "37t"
+                  }
+                ]
+              }
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "200",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "items": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "createdTime": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "createdTime"
+                      },
+                      "externalId": {
+                        "type": "string",
+                        "description": "externalId"
+                      },
+                      "isGlobal": {
+                        "type": "boolean",
+                        "description": "isGlobal"
+                      },
+                      "lastUpdatedTime": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "lastUpdatedTime"
+                      },
+                      "properties": {
+                        "type": "object",
+  
```

Comment:
Please fix propertyValue error 

##[error]The connector category is invalid. Available categories are: [AI, Business Management, Business Intelligence, Collaboration, Commerce, Communication, Content and Files, Finance, Data, Human Resources, Internet of Things, IT Operations, Lifestyle and Entertainment, Marketing, Productivity, Sales and CRM, Security, Social Media, Website]. ValuePath: /x-ms-connector-metadata/2/propertyValue